client/my-sites/stats/stats-purchase/stats-purchase-single-item.tsx

```diff
@@ -496,7 +496,7 @@ function StatsCommercialFlowOptOutForm( {
 					( ! isClassificationInProgress && commercialClassificationLastRunAt > 0 ) ) && (
 					<Button
 						variant="secondary"
-						disabled={ ! supportsOnDemandCommercialClassification && isFormSubmissionDisabled() }
+						disabled={ ! supportsOnDemandCommercialClassification || isFormSubmissionDisabled() }
```

Comment:
Since we don't pass any arguments to `isFormSubmissionDisabled`, could we convert it to a variable?

Reply:
Addressed in 9866155ca2c